src/components/generic/movieModal/style.tsx

```diff
@@ -2,7 +2,7 @@ import styled, {keyframes, css} from 'styled-components'
 import posed from 'react-pose'
 
 // configure animation `react-pose`
-const content = posed.div({
+export const content = posed.div({
```

Comment:
It's best practice to have component names to always start with a capital letter. This applies to all the component declarations below.

Reply:
Done ✅